NEWS.md

```diff
@@ -12,6 +12,21 @@
 were enhanced such that more than one summary variable can be derived, e.g.,
 `AVAL` as the sum and `ADT` as the maximum of the contributing records. (#1792)
 
+- The "joined" functions (`derive_vars_joined()`, `derive_var_joined_exist_flag()`,
+`filter_joined()`, and `event_joined()`) were unified: (#2126)
+    - The `dataset_add` and `filter_add` arguments were added to
+    `derive_var_joined_exist_flag()` and `filter_joined()`.
+    - The `filter` argument was renamed to `filter_join` in
+    `derive_var_joined_exist_flag()` and `filter_joined()`.
+    - The `tmp_obs_nr_var`, the `join_type`, the `first_cond_lower`, and the
+    `first_cond_upper` arguments were added to `derive_vars_joined()`.
+    - In `derive_var_joined_exist_flag()`, `filter_joined()`, and `event_joined()`
+    the `first_cond` argument was renamed to `first_cond_upper` and the
+    `first_cond_lower` argument was added.
+    - In all "joined" functions the `filter_add` argument is applied to the
```

Comment:
```suggestion
    - In all __joined__ functions the `filter_add` argument is applied to the
```

Reply:
maybe some bold or italics would be better?

Reply:
ah well then please ignore.  i have some PTSD from certain characters and CRAN - so always on the lookout for issues :)